docs/troubleshooting/troubleshooting.md

```diff
@@ -48,3 +48,32 @@ deploy.go:55] Post "https://192.168.24.211:32443/api/v1/namespaces": x509: certi
 Cause: Karmada has been installed on the cluster before. `karmada-etcd` uses the `hostpath` mode to mount the local storage. When `karmada-etcd` is uninstalled, residual data exists. We need to delete files in the  defualt directory`/var/lib/karmada-etcd`. If the karmdactl  [--etcd-data](https://github.com/karmada-io/karmada/blob/master/pkg/karmadactl/cmdinit/cmdinit.go#L119) parameter is used, please delete the corresponding directory.
 
 Related Issue: [#1467](https://github.com/karmada-io/karmada/issues/1467), [#2504](https://github.com/karmada-io/karmada/issues/2504).
+
+## karmada-webhook keeps on crashing due to "too many open files"
```

Comment:
It does not only happen to webhook, could be any components

Reply:
> What environment did u do the test? I fix this on ubuntu. I mean will it work on other os, like centos

Add environment then. Not sure it will work on other os. In my environment, only karmada-webhook has this problem. Not sure other components has too.

Reply:
The command works on centos too, I did test